libs/components/src/next/form/datepicker/vl-datepicker.component.ts

```diff
@@ -0,0 +1,338 @@
+    tooltipStyle,
+} from '@domg/govflanders-style/component';
+import datepickerUigStyle from './vl-datepicker.uig-css';
+import { CSSResult, html, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { FormControl, FormControlDefaults } from '../form-control/FormControl';
+
+import { Options } from 'flatpickr/dist/types/options';
+import { Instance } from 'flatpickr/dist/types/instance';
+import flatpickr from 'flatpickr';
+
+import Dutch from 'flatpickr/dist/l10n/nl.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { registerWebComponents } from '@domg-wc/common-utilities';
+import { VlButtonInputAddon, VlIconElement } from '@domg-wc/elements';
+import { live } from 'lit/directives/live.js';
+
+export const DatepickerDefaults = {
+    ...FormControlDefaults,
+    value: '',
+    type: '' as 'range' | 'time' | 'date-time',
+    block: false,
+    readonly: false,
+    format: 'd.m.Y',
+    minDate: '',
+    maxDate: '',
+    minTime: '',
+    maxTime: '',
+    amPm: '',
+    pattern: '',
+};
+
+@customElement('vl-datepicker-next')
+export class VlDatepickerComponent extends FormControl {
+    // Properties
+    private value = DatepickerDefaults.value;
+    private type = DatepickerDefaults.type;
+    private block = DatepickerDefaults.block;
+    private readonly = DatepickerDefaults.readonly;
+    private format = DatepickerDefaults.format;
+    private minDate = DatepickerDefaults.minDate;
+    private maxDate = DatepickerDefaults.maxDate;
+    private minTime = DatepickerDefaults.minTime;
+    private maxTime = DatepickerDefaults.maxTime;
+    private amPm = DatepickerDefaults.amPm;
+    private pattern = DatepickerDefaults.pattern;
+
+    // Variables
+    instance: Instance | null = null;
+    private initialValue = '';
+
+    static {
+        registerWebComponents([VlButtonInputAddon, VlIconElement]);
+    }
+
+    static get styles(): (CSSResult | CSSResult[])[] {
+        return [
+            resetStyle,
+            baseStyle,
+            iconStyle,
+            inputFieldStyle,
+            inputAddonStyle,
+            inputGroupStyle,
+            tooltipStyle,
+            datepickerStyle,
+            datepickerUigStyle,
+        ];
+    }
+
+    static get properties() {
+        return {
+            block: { type: Boolean },
+            readonly: { type: Boolean },
+            value: { type: String, reflect: true },
+            type: { type: String },
+            format: { type: String },
+            minDate: { type: String, attribute: 'min-date' },
+            maxDate: { type: String, attribute: 'max-date' },
+            minTime: { type: String, attribute: 'min-time' },
+            maxTime: { type: String, attribute: 'max-time' },
+            amPm: { type: String, attribute: 'am-pm' },
+            pattern: { type: String },
+        };
+    }
+
+    connectedCallback() {
+        super.connectedCallback();
+
+        // Fix tot <-> tot-en-met
```

Comment:
Mag deze comment weg?

Reply:
Heb deze comment hier expliciet ingelaten, gezien die comment ook in de vorige component erbij stond - blijkbaar is dit de enige uitzondering waarbij localisation moet gepatchd worden. Verwijderd.